jsonschema-hyperschema.xml

```diff
@@ -653,6 +662,84 @@
+                    The value of the "hrefPointers" link description property MUST be
+                    an object.  Each property value in the object MUST be a valid
+                    <xref target="I-D.luff-relative-json-pointer">Relative JSON Pointer</xref>,
+                    which is evaluated relative to the position in the instance from which
+                    <xref target="href">"href"</xref> template variable resolution would
+                    normolly begin.
+                </t>
+                <t>
+                    For each property name in the object that matches a variable name in the
+                    "href" URI Template, the value of that property adjusts the starting position
+                    of variable resolution for that variable.  Properties which do not match
+                    "href" template variable names MUST be ignored.
+                </t>
+                <figure>
+                    <preamble>
+                        Consider the following schema for an n-ary tree using the path expansion
+                        URI Template syntax:
+                    </preamble>
+                    <artwork>
+<![CDATA[{
+    "type": "object",
+    "required": ["id"],
+    "properties": {
+        "id": {"type": "integer", "minimum": 0},
+        "children": {"type": "array", "items": {"$ref": "#"}}
+    },
+    "links": [
+        {
+            "rel": "self",
+            "href": "/trees{/root,parent,id}"
+            "hrefPointers": {
+        }
+    ]
+}]]>
+                    </artwork>
+                    <postamble>
+                        The self links here should always include the root node id,
+                        followed by an immediate parent id if one exists, and always ending
+                        with the id of the node that is the context of the self link.
+                    </postamble>
+                </figure>
+                <figure>
+                    <preamble>
+                        Given the following instance:
+                    </preamble>
+                    <artwork>
+<![CDATA[{
+  "id": 0,
+  "children": [ {
+    "id": 1,
+    "children": [ {
+      "id": 2,
+      "children": [ {
+        "id": 3
+} ] } ] } ] }]]>
+                    </artwork>
+                </figure>
+                <t>
+                    We get the following self links for each node in the instance as identifed
+                    by JSON Pointers:
+                    <list style="hanging">
+                        <t hangText='"" (the root node)'>/trees/0/0</t>
+                        <t hangText='"/children/0/children/0"'>/trees/0/0/1</t>
+                        <t hangText='"/children/0/children/0/children/0"'>/trees/0/1/2</t>
+                        <t hangText='"/children/0/children/0/children/0/children/0"'>/trees/2/3</t>
+                    </list>
+                    omitted, giving us the desired URI with two components after "/trees".  The
```

Comment:
This line is weird and unfinished.

Reply:
Yeah... I thought I'd copied over the example from the issue, but I have no idea what actually happened here.